dracut/30ignition/ignition-files.service

```diff
@@ -16,6 +20,9 @@ Before=initrd-parse-etc.service
 # Network may be required to fetch userdata content.
 After=network.target
 
+# After ignition-disks if it is running
+After=ignition-disks.service
```

Comment:
Might be better to be explicit, though.

Reply:
Yeah, no strong opinion. Though it might mislead readers this is actually what's establishing ordering. Maybe a comment like "This is already guaranteed through initrd-root-fs.target, but just to be explicit." ?